build/configs/artik053/artik053_download.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+###########################################################################
+#
+# Copyright 2016 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+#
+# File   : artik053_download.sh
+# Description : Download script for artik053
+
+# Remember, make is invoked from "os" directory
+
+OS_DIR_PATH=${PWD}
+BUILD_DIR_PATH=${OS_DIR_PATH}/../build
+OUTPUT_BINARY_PATH=${BUILD_DIR_PATH}/output/bin
+
+BOARD_DIR_PATH=${BUILD_DIR_PATH}/configs/artik053
+OPENOCD_DIR_PATH=${BOARD_DIR_PATH}/tools/openocd
+
+SYSTEM_TYPE=`getconf LONG_BIT`
+if [ "$SYSTEM_TYPE" = "64" ]; then
+	OPENOCD_BIN_PATH=${OPENOCD_DIR_PATH}/linux64
+else
+	OPENOCD_BIN_PATH=${OPENOCD_DIR_PATH}/linux32
+fi
+
+COMMAND=openocd
```

Comment:
We don't need to use this variable. OPENOCD_BIN_PATH will be changed by system type.